packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -44,56 +45,68 @@ const columnsArray: any[] = [
     title: 'Name',
     attribute: 'name',
     suffix: '',
+    titleDescription: 'Service Name of a span',
```

Comment:
I think this is not always the case, since the table allows grouping by different criteria.

Reply:
`Tag name by which data is grouped and sub-grouped` how would be this?

Reply:
I suggest not adding a tooltip for this one, and changing `title: 'Group'`